zstacklib/zstacklib/hardware/usb/operations.py

```
@@ -0,0 +1,85 @@
+
+    bus = spec.host_bus or device.bus
+    dev = spec.host_device or device.device
+    cmd = [
+        "virsh",
+        "attach-device",
+        spec.vm_id,
+        "--file",
+        f"/dev/bus/usb/{bus}/{dev}",
+        "--persistent",
+    ]
+    result = subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)